packages/dev/core/src/LibDeclarations/webxr.d.ts

```diff
@@ -1192,6 +1192,9 @@ interface XRFrame {
 
 interface XRWebGLDepthInformation extends XRDepthInformation {
     readonly texture: WebGLTexture;
+
+    readonly textureType: XRTextureType;
+    readonly imageIndex?: number | undefined;

```

Comment:
Isn't this equivalent to:
```suggestion
    readonly imageIndex?: number;
```

Reply:
Surprisingly no, it isn't :-)

`readonly imgIdx?: number` means that it is possible it doesn't exist in the object at all. But if it exists, it is a number. `readonly imgIdx: number | undefined` means that it has to exist, but can be `undefined`. combining them both is also possible, which in general means that you can define the property but set it to be undefined. 
This is the way the webxr typings file is defined because of different implementations supporting different variants - in some cases the key doesn't even exist, in some cases the key is there but the underlying system didn't provide its value, so it is undefined.